packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -963,21 +963,21 @@ export class SchemaFactory<
 	public static readonly requiredRecursive = schemaStaticsPublic.requiredRecursive;
 
 	/**
-	 * A special field which holds a unique identifier for an object node.
+	 * A special readonly field which holds an identifier string for an object node.
 	 * @remarks
-	 * The value of this field, a "node identifier", uniquely identifies a node among all other nodes in the tree.
+	 * The value of this field, a "node identifier", a string which identifies a node (or nodes) among all nodes in the tree.
 	 * Node identifiers are strings, and can therefore be used as lookup keys in maps or written to a database.
-	 * When the node is constructed, the identifier field does not need to be populated.
-	 * The SharedTree will provide an identifier for the node automatically.
-	 * An identifier provided automatically by the SharedTree has the following properties:
-	 * - It is a UUID.
+	 * When the node is constructed, the identifier field does not need to be specified.
+	 * When an identifier is not provided, the SharedTree will generate an identifier for the node automatically.
+	 * The identifier generated by the SharedTree has the following properties:
+	 *
+	 * - It is a UUID which will not collide with other generated UUIDs.
 	 * - It is compressed to a space-efficient representation when stored in the document.
-	 * - A compressed form of the identifier can be accessed at runtime via the `Tree.shortId()` API.
-	 * - It will error if read (and will not be present in the object's iterable properties) before the node has been inserted into the tree.
+	 * - A compressed form of the identifier can be accessed at runtime via the {@link TreeNodeApi.shortId|Tree.shortId()} API.
+	 * - It will error if read (and will not be present in the object's iterable properties) before the node has been inserted into a tree.
 	 *
 	 * However, a user may alternatively supply their own string as the identifier if desired (for example, if importing identifiers from another system).
-	 * In that case, it is up to the user to ensure that the identifier is unique within the current tree - no other node should have the same identifier at the same time.
-	 * If the identifier is not unique, it may be read, but may cause libraries or features which operate over node identifiers to misbehave.
+	 * In that case, if the user requires it to be unique, it is up to them to ensure uniqueness.
 	 * User-supplied identifiers may be read immediately, even before insertion into the tree.
 	 *
 	 * A node may have more than one identifier field (though note that this precludes the use of the `Tree.shortId()` API).
```

Comment:
Might as well update this `Tree.shortId()` to `{@link TreeNodeApi.shortId|Tree.shortId()}`

Reply:
```suggestion
	 * A node may have more than one identifier field (though note that this precludes the use of the {@link TreeNodeApi.shortId|Tree.shortId()} API).
```